### DISCUSSION TITLE
南韓戒嚴又解嚴，到底怎麼了

### DISCUSSION THREAD

**Discussion by CeBru13** [1101-GS4719-C/Python-Community-GS4719-C#2680] [General]
南韓在12/3晚上１１點實施戒嚴，12/4  四點半又宣布解嚴，一天之內戒嚴又解嚴，太突然了吧

**Comment by forestdew507** [comment]
敏迪這篇寫得很精彩 ! 看到南韓戒嚴的標題突然想到早上看到的文章，順帶將這個我覺得不錯的自媒體分享給大家 > https://www.instagram.com/p/DDI5Di8zLAD/?utm_source=ig_web_copy_link&igsh=MzRlODBiNWFlZA==

**Comment by kzx-creat** [comment]
笑死，一日體驗戒嚴加解嚴，韓劇都不敢這樣演。尹 還真我行我素

**Comment by Elaine4488** [comment]
預計1月初要去韓國，結果突然看到新聞報導說韓國宣布戒嚴，真的超級緊張，整個人都慌了。畢竟機票和住宿早就訂好了，所有的計劃和準備都已經安排妥當，沒想到馬上解嚴，真的是電視劇都不敢這樣演。
